src/html/components/question.html

```diff
@@ -0,0 +1,8 @@
+<div class="question" id="question">
+  <h2 class="section-title">The Question</h2>
+  <p class="section-intro">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</p>
+  <img src="images/phone.svg" alt="" class="phone">
+  <p>Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<sup><a href="">4</a></sup> Smartphones offer many additional features but impose a barrier of entry that seemed too high. Reliance on personal computers is also problematic, especially among disadvantaged patients.</p>
+  <p>We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<sup><a href="">5</a></sup> Furthermore, adherence studies using text reminders show promising results.<sup><a href="">6</a></sup>  Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<sup><a href="">7</a></sup></p>
```

Comment:
One pattern we like to do for things that have yet to be completed is to add a `TODO` note. In the case of this anchor (`<a href="">`) that would be `<a href="#TODO">`. That way you can later do search for "TODO" to catch anything you might have forgot to complete.

Other examples would be regular html or css comments:
```
<!-- TODO: wire-up to database -->
/* TODO add a fallback */
```